doc/user_guide/encryption.rst

```diff
@@ -9,10 +9,11 @@ Exasol published a few articles describing the details:
 - `CHANGELOG: TLS for all Exasol drivers <https://exasol.my.site.com/s/article/Changelog-content-6507>`_
 - `CHANGELOG: Database accepts only TLS connections <https://exasol.my.site.com/s/article/Changelog-content-16927>`_
 - `Generating TLS files yourself to avoid providing a fingerprint <https://exasol.my.site.com/s/article/Generating-TLS-files-yourself-to-avoid-providing-a-fingerprint/>`_
+- `TLS connection fails <https://exasol.my.site.com/s/article/TLS-connection-fails>`_
 
 Default
 -------
-Encryption with certification validation is ENABLED by default starting from PyExasol version ``1.0.0``.
+Encryption with certification validation is ENABLED by default starting from PyEXASOL version ``1.0.0``.
```

Comment:
we really have too many ways of writing pyexasol, the official documentation seems to use PyExasol https://docs.exasol.com/db/latest/connect_exasol/drivers/python/pyexasol.htm?Highlight=Pyexasol, but let me double-check.

Reply:
:heavy_check_mark: updated references to PyExasol as per chat.